js/app.js

```diff
@@ -0,0 +1,33 @@
+let newShoppingList = new List();
+
+document.getElementById('addItemForm').onsubmit = function (event) {
+  event.preventDefault()
+  addToShoppingList(event);
+};
+
+function addToShoppingList(e) {
+  let itemName = e.target[0].value;
+  let itemDescription = e.target[1].value;
+  let newShoppingListItem = new Item(itemName, itemDescription);
+
+  newShoppingList.addItem(newShoppingListItem);
+  reRender();
+}
+function removeItemButtonClicked(idx) {
+  newShoppingList.removeItem(newShoppingList.items[idx]);
+  reRender();
+}
+function reRender() {
+  let htmlList = newShoppingList.render()
+  let content = document.getElementById('content');
+  content.innerHTML = htmlList;
+}
+function changeCheckedStatus(idx, checkBox) {
```

Comment:
Make sure there are line breaks in between your functions. It makes it easier to read what's going on.